CreateSnapshot/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'application'
+    id 'java'
+    id 'jacoco'
+    id 'io.freefair.lombok' version '8.6'
+}
```

Comment:
Testing comment here.

Reply:
Created a JIRA task: https://opensearch.atlassian.net/browse/MIGRATIONS-1819

Reply:
Created a JIRA task: [](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA task: [](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA task: [](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA task for the comment above: [This is a great title](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [title](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [Jira Title Here](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [I want to be notified](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [New Comment](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [Send me straight to this comment](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [To the real link this time?](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [zyefsafd](https://opensearch.atlassian.net/browse/MIGRATIONS-1819)

Reply:
Created a JIRA issue: [This is a real task created from a JIRA issue.](https://opensearch.atlassian.net/browse/MIGRATIONS-1828)

Reply:
Created a JIRA issue: [This is my Jira Task Title](https://opensearch.atlassian.net/browse/MIGRATIONS-1829)

Reply:
Created a JIRA issue: [Shoudl have a direct link to the founding comment](https://opensearch.atlassian.net/browse/undefined)

Reply:
Created a JIRA issue: [Should have a direct link to the founding comment](https://opensearch.atlassian.net/browse/undefined)

Reply:
Created a JIRA issue: [This is try approximately 3000](https://opensearch.atlassian.net/browse/MIGRATIONS-1832)